renderdoc/driver/metal/metal_command_buffer_bridge.mm

```diff
@@ -187,6 +187,13 @@ - (NSError *)error
   return id<MTLBlitCommandEncoder>(GetWrapped(self)->blitCommandEncoder());
 }
 
+- (nullable id<MTLBlitCommandEncoder>)blitCommandEncoderWithDescriptor:
```

Comment:
Small thing which will come when the next round of PRs come but this change looks like a noise change, the function hasn't changed implementation and has just moved in the file.